### PR TITLE
[Backup] [InProgress] SAP Hana container register fix

### PR DESCRIPTION
**Pull request by sambitratha**
**Description**

**Testing Guide**

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Backup

File: src/azure-cli/azure/cli/command_modules/backup/custom_wl.py [hunk 1]
```diff
@@ -101,20 +101,18 @@ def register_wl_container(cmd, client, vault_name, resource_group_name, workload
...
-    if not cust_help.is_native_name(container_name):
-        raise CLIError(
-            """
-            Container unavailable or already registered.
-            """)
+        if container_name is None or not cust_help.is_native_name(container_name):
+            raise CLIError(
+                """
+                Container unavailable or already registered.
+                """)
```
Comment (zhoxing-ms) [inline src/azure-cli/azure/cli/command_modules/backup/custom_wl.py:115]: Please the specific error type `ArgumentUsageError` instead of `CLIError`

**Comment by zhoxing-ms** [comment]
@sambitratha May I ask what does the `[InProgress]` in PR title mean?

**Comment by zhoxing-ms** [comment]
@sambitratha Could you please address those conflicts?

**Comment by zhoxing-ms** [comment]
@sambitratha Please solve these comments and conflicts in time today, otherwise it will miss the release of this sprint and have to release it in the next sprint (about 08-06)

**Comment by zhoxing-ms** [comment]
@sambitratha Any update?

**Comment by zhoxing-ms** [comment]
Please solve these comments and conflicts in time today, otherwise this PR will miss the release of this sprint and can only be released in the next sprint (09-07)

**Comment by zhoxing-ms** [comment]
Since I haven't received any reply for a long time, so I close it first. If we still need merge it in the future, then we can reopen it

**Pull request closed without merging**